pkg/status/health/health_test.go

```diff
@@ -13,17 +13,28 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestEmptyCatalog(t *testing.T) {
+	cat := newCatalog()
+
+	status := cat.getStatus()
+	assert.Len(t, status.Healthy, 0)
+	assert.Len(t, status.Unhealthy, 0)
+}
+
 func TestCatalogStartsHealthy(t *testing.T) {
 	cat := newCatalog()
+	_ = cat.register("test1")
 
 	status := cat.getStatus()
 	assert.Len(t, status.Healthy, 1)
 	assert.Contains(t, status.Healthy, "healthcheck")
-	assert.Len(t, status.Unhealthy, 0)
+	assert.Len(t, status.Unhealthy, 1)
+	assert.Contains(t, status.Unhealthy, "test1")
 }
 
 func TestCatalogGetsUnhealthyAndBack(t *testing.T) {
 	cat := newCatalog()
+	_ = cat.register("test1")
```

Comment:
Not sure why you had to register this, could you explain? To make sure that the healthcheck becomes unhealthy?

Reply:
I added a comment in the code.

Previously, when no component were registered, the `healthcheck` check would still be verified but, as no component is ever probed, this check would eventually fail.

So, I did a change so that, when no component is registered, the `healthcheck` check isn’t verified any more.
As the purpose of this unit test is to validate the behavior of the `healthcheck` check, I need a fake component so that it isn’t completely disabled.